org.unofficialrevport.REVHubInterface.yml

```diff
@@ -0,0 +1,69 @@
+id: org.unofficialrevport.REVHubInterface
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: REVHubInterface.sh
+finish-args:
+  - --device=all
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --talk-name=org.freedesktop.Flatpak 
+modules:
+  - python3-requirements.json
+  - name: tkinter
+    buildsystem: simple
+    build-commands:
+      - pip3 install --prefix=${FLATPAK_DEST} .
+    sources:
+      - type: git
+        url: https://github.com/iwalton3/tkinter-standalone
+        commit: 88aa05075d90d393a29a484bce676e237d311082
+    modules:
+      - name: tcl
+        buildsystem: autotools
+        subdir: unix
+        post-install:
+          - chmod 755 /app/lib/libtcl*.so
+        cleanup:
+          - /bin
+          - /lib/pkgconfig
+          - /man
+        sources:
+          - type: archive
+            url: https://prdownloads.sourceforge.net/tcl/tcl8.6.14-src.tar.gz
+            sha256: 5880225babf7954c58d4fb0f5cf6279104ce1cd6aa9b71e9a6322540e1c4de66
+            x-checker-data:
+              type: anitya
+              project-id: 4941
+              stable-only: true
+              url-template: https://prdownloads.sourceforge.net/tcl/tcl$version-src.tar.gz
+      - name: tk
+        buildsystem: autotools
+        subdir: unix
+        post-install:
+          - chmod 755 /app/lib/libtk*.so
+        cleanup:
+          - /bin
+          - /lib/pkgconfig
+          - /man
+        sources:
+          - type: archive
+            url: https://prdownloads.sourceforge.net/tcl/tk8.6.14-src.tar.gz
+            sha256: 8ffdb720f47a6ca6107eac2dd877e30b0ef7fac14f3a84ebbd0b3612cee41a94
+            x-checker-data:
+              type: anitya
+              project-id: 11426
+              stable-only: true
+              url-template: https://prdownloads.sourceforge.net/tcl/tk$version-src.tar.gz
+  - name: revhubinterface
+    buildsystem: simple
+    builddir: true
+    build-commands:
+      - pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} . --no-build-isolation
+      - install -D REVHubInterface.sh /app/bin/REVHubInterface.sh
+    sources:
+      - type: git
+        url: https://github.com/unofficial-rev-port/REVHubInterface.git
+        branch: main
```

Comment:
NEVER a git branch

Reply:
Should I remove branch?

Reply:
Removed

Reply:
you need commit or a tag.

Reply:
If I pin a specific commit or tag version, then how will I be able to update in future, will I need to make a pr for every new version?

Reply:
that's how it works. Ideally it should be a tag / release because it's shipped.

Reply:
So a contributor would need to make a pr to flathub just like this one for every release? I don't really know how this works, first time

Reply:
https://github.com/flathub/flathub/pull/5224#discussion_r1594873315